### PR TITLE
[ADD] new addon website_blog_facebook_comment

### PR DESCRIPTION
**Pull request by Endika**
# Blog with Facebook comment

This module add facebook comments in blog post.

### CODE REVIEW

File: website_blog_facebook_comment/README.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+============
+
+To install this module, you need to:
+
+* website_blog
+
+Configuration
+=============
+
+To configure this module, you need to:
```
Comment (pedrobaeza) [inline website_blog_facebook_comment/README.rst:19]: Where?

File: website_blog_facebook_comment/__init__.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Python source code encoding : https://www.python.org/dev/peps/pep-0263/
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
```
Comment (pedrobaeza) [inline website_blog_facebook_comment/__init__.py]: Remove this line

File: website_blog_facebook_comment/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Python source code encoding : https://www.python.org/dev/peps/pep-0263/
+##############################################################################
+#
+#    OpenERP, Odoo Source Management Solution
```
Comment (pedrobaeza) [inline website_blog_facebook_comment/__openerp__.py]: Remove this line

File: website_blog_facebook_comment/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': "Blog with Facebook comment",
```
Comment (pedrobaeza) [inline website_blog_facebook_comment/__openerp__.py]: Better to put `Add Facebook comments on blog posts`

File: website_blog_facebook_comment/controllers/__init__.py [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Python source code encoding : https://www.python.org/dev/peps/pep-0263/
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
```
Comment (pedrobaeza) [inline website_blog_facebook_comment/controllers/__init__.py]: Remove this line in all places

File: website_blog_facebook_comment/controllers/main.py [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+                  tag_id=None, page=1, enable_editor=None, **post):
+        response = super(WebsiteBlog, self).blog_post(
+            blog, blog_post, tag_id=None, page=1, enable_editor=None, **post)
+        response.qcontext['appId'] = request.website.facebook_appid
+        response.qcontext['lang'] = request.context['lang']
+        response.qcontext['numposts'] = request.website.facebook_numposts
+        base_url = (self._base_url() + '/blog/' +
+                    str(slug(response.qcontext['blog'])) + '/post/' +
+                    str(slug(response.qcontext['blog_post'])))
+        response.qcontext['base_url'] = base_url
```
Comment (pedrobaeza) [inline website_blog_facebook_comment/controllers/main.py]: For what is this needed? You can obtain the current full URL querying request variable

File: website_blog_facebook_comment/views/facebook_settings_view.xml [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+<openerp>
+    <data>
+
+        <record id="view_facebook_comment_configuration" model="ir.ui.view">
+            <field name="name">Facebook comment settings</field>
+            <field name="model">website.config.settings</field>
+            <field name="inherit_id" ref="website.view_website_config_settings"/>
+            <field name="arch" type="xml">
+                <xpath expr="//div[@name='social_facebook']" position="after">
+                      <label for="id" string="Facebook comment API"/>
```
Comment (pedrobaeza) [inline website_blog_facebook_comment/views/facebook_settings_view.xml]: Bad indentation or tabs.

File: website_blog_facebook_comment/views/blog_post_complete_view.xml [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+<openerp>
+<data>
+
+    <template id="blog_post_complete" name="Blog Post" inherit_id="website_blog.blog_post_complete">
+        <xpath expr="//section[@id='comments']" position="attributes">
+            <attribute name="style">display: none;</attribute>
+        </xpath>
+
+        <xpath expr="//section[@id='comments']" position="after">
+            <div id="comments_facebook" class="read_width">
```
Comment (pedrobaeza) [inline website_blog_facebook_comment/views/blog_post_complete_view.xml]: Encapsulate this in a t-if clause in case you don't have Facebook ID set.

**Comment by Endika** [comment]
Ok, @pedrobaeza

**Comment by pedrobaeza** [comment]
:+1:

**Comment by Endika** [comment]
I using CSS to hide the comment text

**Comment by rafaelbn** [comment]
We are using this module in production for near 2 months an it's Working fine.

@pedrobaeza why travis fails?

**Comment by pedrobaeza** [comment]
Travis fail is unrelated due to website_sale_\* modules.

**Comment by rafaelbn** [comment]
We are using this module in production enviroment for 2 months. No issues. :+1:

**Comment by rafaelbn** [comment]
Hi @eLBati , could you check this PR please?

**Comment by JavierIniesta** [comment]
It is operating in production environment.
:+1:

**Comment by pedrobaeza** [comment]
Merging also.

**Pull request merged**